pipeline/07-export.R

```diff
@@ -541,9 +570,14 @@ for (town in unique(assessment_pin_prepped$township_code)) {
   addStyle(
     wb, comp_sheet_name,
     style = style_comma,
-    rows = comp_row_range, cols = c(13, 15), gridExpand = TRUE
+    rows = comp_row_range, cols = c(14, 16), gridExpand = TRUE
```

Comment:
Having to do all of this manual column realignment has been giving me the itch to refactor this pipeline stage to use a unified data structure representing the format of the sheets, so that all of these references to column positions and formats can derive them from a single source of truth... I think it would take a day or two to complete, since it would be a big refactor and would require lots of QC to make sure we don't mess anything up, but if you're aligned I can open a backburner issue for it.

Reply:
Sure! I think it'd be worthwhile given how often we export stuff to excel.